modules/setup-bias-monitoring.adoc

```diff
@@ -0,0 +1,83 @@
+:_module-type: PROCEDURE
+
+[id="setup-bias-monitoring_{context}"]
+= Before you begin
+
```

Comment:
Add introductory text to the "Before you begin" section

Also consider renaming the "Before you begin" section to something like:
"Setting up your working environment"

Before you deploy the models, complete the following set up tasks:

- Download the tutorial files
- Enable user workload monitoring
- Enabling the TrustyAI component
- Create a project
- Authenticate the TrustyAI service